plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/manager/VmwareManagerImpl.java

```diff
@@ -1180,10 +1180,7 @@ public void doInTransactionWithoutResult(TransactionStatus status) {
 
     private void validateZone(Long zoneId) throws InvalidParameterValueException {
```

Comment:
No, this method is being used in other part of code. This method does more than zone existence check.